channels/applicationinsights-channel-js/src/Sender.ts

```diff
@@ -590,6 +634,8 @@ export class Sender extends BaseTelemetryPlugin implements IChannelControlsAI {
                         _self._appId = response.appId;
                     }
                 }
+                //let isOrgEndpoint = responseUrl !== _orgEndpointUrl;
+                let isOrgEndpoint = true;
```

Comment:
Lets not worry about this case and just document as a "potential" issue that when changing the endpoint if a retry is required that any previous events may (if not dropped) be sent to the new endpoint.

Basically, this is where we say that it you want the events to only be sent to the original endpoint that you should either
- Create a 2nd instance and only use that instance
- Flush and wait for the send to be successful (not sure if we currently has a callback on the flush that returns only after all events are flushed -- we do for 1DS).